client/src/components/Navbar/Navbar.jsx

```diff
@@ -8,85 +8,251 @@ import Stack from '@mui/material/Stack';
 import Button from '@mui/material/Button';
 import EmailIcon from '@mui/icons-material/Email';
 import { Link } from 'react-router-dom';
+import SearchIcon from '@mui/icons-material/Search';
+import AccountCircleIcon from '@mui/icons-material/AccountCircle';
+import IconButton from '@mui/material/IconButton';
+import { alpha } from '@mui/material/styles';
+import InputBase from '@mui/material/InputBase';
+import theme from '../../theme';
+import { useState } from 'react';
+import { Divider, Menu, MenuItem, Tooltip } from '@mui/material';
 
-const Navbar = () => (
-	<AppBar
-		sx={{
-			boxShadow: 0,
-			position: 'static',
-		}}>
-		<Container
-			maxWidth="md"
-			sx={{ position: 'relative' }}>
-			<Toolbar
-				sx={{
-					display: 'flex',
-					alignItems: 'center',
-					justifyContent: 'space-between',
-					flexShrink: 0,
-				}}>
-				<Box>
-					<Typography
-						variant="h4"
-						fontWeight={200}>
-						shelf
-						<span style={{ fontWeight: 'bold' }}>share</span>
-					</Typography>
-				</Box>
-			</Toolbar>
-			{/* Login/SignUp Card Large Screen*/}
-			<Card
-				variant="outlined"
-				sx={{
-					display: { xs: 'none', sm: 'block' },
-					position: 'absolute',
-					right: '0',
-					top: '0',
-					mt: 2,
-					mr: 2,
-					width: '300px',
-					padding: 2,
-					zIndex: 1,
-				}}>
-				<Stack
-					direction={'column'}
-					spacing={2}
-					justifyContent={'center'}
-					alignItems={'center'}>
-					<Typography variant="h6">Discover & read more</Typography>
-					<Button
-						variant="outlined"
-						component={Link}
-						to="/signup"
-						size="large"
-						sx={{ width: '80%', mt: 3 }}
-						startIcon={<EmailIcon fontSize="small"></EmailIcon>}>
+const Navbar = () => {
+	const [anchorEl, setAnchorEl] = useState(null);
+	// Needs to be changed for getting if a user is logged in or not
+	const isAuthenticated = true;
```

Comment:
Are you saying we need to grab isAuthenticated from the store? I think either you or Alex were able to do this for the Login page using 
`const user = useSelector(state => state.session.userInfo);`
<img width="896" alt="image" src="https://github.com/chingu-voyages/v49-tier3-team-29/assets/72801181/7b591b6d-2ef9-42b1-aab1-2723a3de9ea2">

Reply:
Correct, this functionality was added after this PR was made

Reply:
Sounds good to me